packages/components/constraints/src/helpers.ts

```diff
@@ -1,12 +1,15 @@
 import { customProperties as vars } from '@commercetools-uikit/design-system';
 
-const getMaxPropTokenValue = (max) => {
+// `null` is derived from `getMaxPropEquivalent`
+const getMaxPropTokenValue = (max: number | string | null) => {
+  if (!max) return null;
+  // @ts-expect-error
```

Comment:
What is the error?

Reply:
I think it's computed property name.
To be more type safe we would need to have a switch statement or something.
It can also be a follow up.

Reply:
it's the computed property. I described in L3 where it comes from ;)